### PR TITLE
add frbs + gcmr extensions to recent R-bundle-CRAN easyconfigs

### PR DESCRIPTION
**Pull request by pavelToman**
(created using `eb --new-pr`)
resolves https://github.com/vscentrum/vsc-software-stack/issues/471

add  `gcmr` to 2023a bundle as dependency of `scNanoGPS` for https://github.com/vscentrum/vsc-software-stack/issues/467

### CODE REVIEW

**Comment by pavelToman** [comment]
@boegelbot please test @ generoso

**Comment by pavelToman** [comment]
Test report by @pavelToman
**FAILED**
Build succeeded for 1 out of 2 (2 easyconfigs in total)
node4012.donphan.os - Linux RHEL 8.8, x86_64, Intel(R) Xeon(R) Gold 6240 CPU @ 2.60GHz, 1 x NVIDIA NVIDIA A2, 545.23.08, Python 3.6.8
See https://gist.github.com/pavelToman/b080793b5f1c170fa42a174cc4d7fa37 for a full test report.

**Comment by pavelToman** [comment]
@boegelbot please test @ jsc-zen3

**Comment by boegel** [comment]
> Test report by @pavelToman **FAILED** Build succeeded for 1 out of 2 (2 easyconfigs in total) node4012.donphan.os - Linux RHEL 8.8, x86_64, Intel(R) Xeon(R) Gold 6240 CPU @ 2.60GHz, 1 x NVIDIA NVIDIA A2, 545.23.08, Python 3.6.8 See https://gist.github.com/pavelToman/b080793b5f1c170fa42a174cc4d7fa37 for a full test report.

@pavelToman What happened here? Maybe you have something installed in your home directory?

```
Error: renv 1.0.3 not installed, found 1.0.7.9000 instead
```

**Review by boegel — APPROVED** [review]
lgtm

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in total)
node3102.skitty.os - Linux RHEL 9.4, x86_64, Intel(R) Xeon(R) Gold 6140 CPU @ 2.30GHz (skylake_avx512), Python 3.9.18
See https://gist.github.com/boegel/94e02ede89f792388fa2f49bab937908 for a full test report.

**Comment by pavelToman** [comment]
> > Test report by @pavelToman **FAILED** Build succeeded for 1 out of 2 (2 easyconfigs in total) node4012.donphan.os - Linux RHEL 8.8, x86_64, Intel(R) Xeon(R) Gold 6240 CPU @ 2.60GHz, 1 x NVIDIA NVIDIA A2, 545.23.08, Python 3.6.8 See https://gist.github.com/pavelToman/b080793b5f1c170fa42a174cc4d7fa37 for a full test report.
> 
> @pavelToman What happened here? Maybe you have something installed in your home directory?
> 
> ```
> Error: renv 1.0.3 not installed, found 1.0.7.9000 instead
> ```

Yes, problem was in my HOME. Gonna run it again.

File: easybuild/easyconfigs/r/R-bundle-CRAN/R-bundle-CRAN-2023.12-foss-2023a.eb [hunk 1]
```diff
@@ -3430,6 +3430,12 @@ exts_list = [
     ('missMDA', '1.19', {
         'checksums': ['f9675884829b2fef75237c335b21991d163674320e766523c71c7a853f95e65c'],
     }),
+    ('gcmr', '1.0.3', {
```
Comment (smoors) [inline easybuild/easyconfigs/r/R-bundle-CRAN/R-bundle-CRAN-2023.12-foss-2023a.eb:3433]: @pavelToman why is `gcmr` added here and not to the foss-2023b one?

Comment (pavelToman) [reply easybuild/easyconfigs/r/R-bundle-CRAN/R-bundle-CRAN-2023.12-foss-2023a.eb:3433]: add gcmr to 2023a bundle as dependency of scNanoGPS for https://github.com/vscentrum/vsc-software-stack/issues/467
Should I add it either to 2023b? I need it for 2023a

Comment (smoors) [reply easybuild/easyconfigs/r/R-bundle-CRAN/R-bundle-CRAN-2023.12-foss-2023a.eb:3433]: yes, better to add it to 2023b too

**Comment by pavelToman** [comment]
@boegelbot please test @ generoso

**Comment by verdurin** [comment]
@boegelbot please test @ jsc-zen3

**Comment by verdurin** [comment]
Should have added `--skip` to the test command...

**Review by boegel — APPROVED** [review]
lgtm

**Comment by boegel** [comment]
Going in, thanks @pavelToman!

**Pull request merged**